src/backend/gl/src/queue.rs

```diff
@@ -62,8 +63,12 @@ pub struct CommandQueue {
     features: hal::Features,
     vao: Option<native::VertexArray>,
     state: State,
+    presentation_fence: <glow::Context as glow::HasContext>::Fence,
```

Comment:
let's use `Starc` as we do with other native stuff here, for consistency, and to avoid `unsafe impl` that follow